src/scripts/danish-legal/create_completeness_detection.py

```diff
@@ -0,0 +1,299 @@
+"""Create the legal completeness detection dataset."""
+
+import logging
+import random
+import re
+from pathlib import Path
+from typing import Dict, List, Set
+
+import click
+import pandas as pd
+from datasets import Dataset, DatasetDict, Split
+from docling.document_converter import DocumentConverter
+from huggingface_hub import HfApi
+from requests import HTTPError
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# Constants
+COMPLETE_CONTRACT_PROBABILITY = 0.3
+NON_REQUIRED_REMOVAL_PROB = 0.3  # For complete contracts
+OPTIONAL_REMOVAL_PROB = 0.4  # For incomplete contracts
```

Comment:
Shouldn't these two just be merged, as it is the probability of removing an optional section, no matter whether the the contract is complete or not?